index.html

```diff
@@ -33,18 +34,177 @@ <h1 class="presentation">I'm Mehdi jdir
+                     <li class="icon"><a href="" target="_blank"><img src="./img/Linkedin.png" alt="Linkedin link"></a></li>
+                     <li class="icon"><a href="" target="_blank"><img src="./img/Mobile.png" alt="Mobile link"></a></li>
+                     <li class="icon"><a href="" target="_blank"><img src="./img/github.png" alt="github link"></a></li>
+                     <li class="icon"><a href="" target="_blank"><img src="./img/succes.png" alt="succes link"></a></li>
                                          
                    </ul>
                 </div>
             
         </section>
       </div>
+      <section class="gridsec">
+      <div class="page1">
+        
+          <div class="image1"><img src="./img/Snapshoot Portfolio.png" alt="Snapshoot image1"</div>
+        
+        <section class="sec">
+          <h6 class="text1-1">Tonic</h6>
+          <Div class="list">
+            <div class="can">CANOPY</div>
+            <div class="dot"></div>
+            <div class="Bac">Back End Dev </div>
+            <div class="dot"></div>
+            <div class="Bac">2015</div>
+         </Div>
+          <p class="par">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="but">
+            <li> <input class="button" type="button" value="HTML"></li>
+            <li> <input class="button" type="button" value="CSS"></li>
+            <li> <input class="button" type="button"  value="JavaScript"></li>
+          </ul>
+          <div class="see">
+            <input class="btngr" type="button" value="See project">
+          </div>
+        </section>
+      </div>
+
+
+
+      <div class="page1">
+          <div class="image1"><img src="./img/image2.png" alt="Snapshoot image2"</div>
+        <section class="sec">
+          <h6 class="text1-1">Multi-Post
+            Stories</h6>
+          <Div class="list">
+            <div class="can">CANOPY</div>
+            <div class="dot"></div>
+            <div class="Bac">Back End Dev </div>
+            <div class="dot"></div>
+            <div class="Bac">2015</div>
+         </Div>
+          <p class="par">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="but">
+            <li> <input class="button" type="button" value="HTML"></li>
+            <li> <input class="button" type="button" value="CSS"></li>
+            <li> <input class="button" type="button"  value="JavaScript"></li>
+          </ul>
+          <div class="see">
+            <input class="btngr" type="button" value="See project">
+          </div>
+        </section>
+      </div>
+
+
+
+      <div class="page1">
+        
+          <div class="image1"><img src="./img/image3.png" alt="Snapshoot image3"</div>
+       
+        <section class="sec">
+          <h6 class="text1-1">Tonic</h6>
+          <Div class="list">
```

Comment:
- ***[OPTIONAL]*** 
- In HTML, the correct tag for a division or container element is `<div>`, not `<Div>`. HTML tags are case-insensitive, which means that you can write them in uppercase, lowercase, or a mix of both. However, it is considered best practice to use lowercase letters for HTML tags. 

- [ ] Kindly follow the best practice everywhere.